traffic_ops/testing/api/v4/deliveryservices_test.go

```diff
@@ -1035,11 +1037,54 @@ func UpdateTestDeliveryServices(t *testing.T) {
 	updatedLongDesc := "something different"
 	updatedMaxDNSAnswers := 164598
 	updatedMaxOriginConnections := 100
+	updatedActive := false
+	//updatedCdnId : =
+	updatedDisplayName := "newds1displayname"
+	updatedDscp := 41
+	updatedGeoLimit := 1
+	updatedInitialDispersion := 2
+	updatedIpv6RoutingEnabled := false
+	updatedLogsEnabled := false
+	updatedMissLat := 42.881944
+	updatedMissLong := -88.627778
+	updateMultisiteOrigin := true
+	updatedOrgServerFqdn := "http://origin.example.net"
+	//updatedProfile :=
```

Comment:
should this be commented out? If so, do we need it in here at all?

Reply:
Added the validation for profiles.